### ISSUE TITLE
Error in training

### ISSUE THREAD

**Issue by Itamar651** [pperle/gaze-tracking#3]
I got error when running train.py (which should train the model according to what I understood) the error was related to the import of EPOCH_OUTPUT from the pytorch_lightning.utilities.types library.

Do you know why?
Do you know how to solve this error?
Where I was wrong??

Thanks :)

**Comment by glowfisch8lan** [comment]
Yes, the error occurs because the version of pytorch_lighting and torchmetrics must be strictly.
torchmetrics==0.7.1 
pytorch_lightning==1.4.9
